### PR TITLE
Add support for bluetooth presentment

### PR DESCRIPTION
**Pull request by w4ll3**

### CODE REVIEW

**Comment by tristanmiller-spruceid** [comment]
I created a task to cover some cleanup tasks, but we should get this in and starting to use it with react-native.

**Review by tristanmiller-spruceid — APPROVED** [review]

File: WalletSdk/src/main/java/com/spruceid/walletandroidsdk/BleCentral.kt [hunk 1]
```diff
@@ -0,0 +1,156 @@
+package com.spruceid.walletandroidsdk
```
Comment (sbihel) [inline WalletSdk/src/main/java/com/spruceid/walletandroidsdk/BleCentral.kt:1]: To be consistent with the other libraries I think it should be `com.spruceid.wallet.sdk:walletsdk`

**Review by sbihel — CHANGES_REQUESTED** [review]
Maybe it's due to the move to maven central but when I tested it it didn't get past the qr code -- and from the logs there doesn't seem to be any activity past "start scanning". Does it work for you locally?

**Comment by sbihel** [comment]
I had simply denied the "Discover Nearby Devices" permission 🤦

**Review by sbihel — APPROVED** [review]

**Pull request merged**